Dockerfile

```diff
@@ -71,6 +71,11 @@ RUN cd /usr/local/bin && \
     wget --quiet https://get.sensiolabs.org/security-checker.phar && \
     chmod +x security-checker.phar
 
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+ENV PATH=/root/.cargo/bin:$PATH
```

Comment:
The `/root` part confuses me, but apparently `$HOME` is not accessible in Dockerfile. Couldn't find a way around it

Reply:
what about `/usr/local/opt/`?